SoftLayer/CLI/hardware/sensor.py

```diff
@@ -0,0 +1,69 @@
+"""Hardware internal Sensor ."""
+# :license: MIT, see LICENSE for more details.
+
+import SoftLayer
+from SoftLayer.CLI import click
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--discrete', is_flag=True, default=False, help='Show discrete units associated hardware sensor')
+@environment.pass_env
+def cli(env, identifier, discrete):
+    """Retrieve a server’s hardware state via its internal sensors."""
+    mgr = SoftLayer.HardwareManager(env.client)
+    sensors = mgr.get_sensors(identifier)
+
+    temperature_table = formatting.Table(["sensor", "status", "Reading", "min", "Max"],
+                                         title='temperature Degree c')
+
+    volts_table = formatting.Table(["sensor", "status", "Reading", "min", "Max"],
+                                   title='volts')
+
+    watts_table = formatting.Table(["sensor", "status", "Reading"],
+                                   title='Watts')
+
+    rpm_table = formatting.Table(["sensor", "status", "Reading", "min"],
+                                 title='RPM')
+
+    discrete_table = formatting.Table(["sensor", "status", "Reading"],
+                                      title='discrete')
```

Comment:
Make sure all the table columns and titles start with a capital letter.
```suggestion
    temperature_table = formatting.Table(["Sensor", "Status", "Reading", "Min", "Max"],
                                         title='Temperature (C)')

    volts_table = formatting.Table(["Sensor", "Status", "Reading", "Min", "Max"],
                                   title='Volts')

    watts_table = formatting.Table(["Sensor", "Status", "Reading"],
                                   title='Watts')

    rpm_table = formatting.Table(["Sensor", "Status", "Reading", "Min"],
                                 title='RPM')

    discrete_table = formatting.Table(["Sensor", "Status", "Reading"],
                                      title='Discrete')
```